ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml

```diff
@@ -73,6 +67,11 @@ chain:
           oc get clusterversion 2>/dev/null || true
           sleep 1s
       done
+      
```

Comment:
This step was only needed for our cluster-bot. If you need guarantees for this it should actually check ` hcluster.Status.KubeadminPassword`